app/shared/test/services/settings.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+SECRET_KEY = os.environ.get("SECRET_KEY")
+
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "shared",
+]
+
+DATABASES = {
+    "default": {
+        "ENGINE": os.environ.get("COCONNECT_DB_ENGINE"),
```

Comment:
Same comment as above - we should rename the `COCONNECT_*` to `CARROT_*` or similar. To worry about later!

Reply:
Agreed - one for the backlog